base-images-sbom-script/base_image_sbom_script.py

```diff
@@ -0,0 +1,86 @@
+import json
+import argparse
+import pathlib
+
+from packageurl import PackageURL
+
+
+def get_base_images_sbom_data(images, is_last_from_scratch):
+
+    new_data = {"formulation": [{"components": []}]}
+
+    for index, image in enumerate(images):
+        property_name = f"builder-{index}"
```

Comment:
You can use the same property name multiple times, there's no need to have the index in the name (that would make the property a bit harder to process)

Maybe `is_builder_image:for_stage`

Reply:
updated